<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StuDialog</class>
 <widget class="QDialog" name="StuDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>341</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="1">
     <widget class="QLineEdit" name="AddEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="StuSex">
      <property name="text">
       <string>性别：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="NameEdit"/>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="SexBox"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="IDEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="StuName">
      <property name="text">
       <string>姓名：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="StuBirth">
      <property name="text">
       <string>年龄：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="StuID">
      <property name="text">
       <string>患者ID：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="StuHome">
      <property name="text">
       <string>电话：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="HomeEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="StuAdd">
      <property name="text">
       <string>诊断结果：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="BirthEdit"/>
    </item>
   </layout>
   <zorder>StuID</zorder>
   <zorder>StuName</zorder>
   <zorder>StuSex</zorder>
   <zorder>StuBirth</zorder>
   <zorder>StuAdd</zorder>
   <zorder>StuHome</zorder>
   <zorder>HomeEdit</zorder>
   <zorder>AddEdit</zorder>
   <zorder>NameEdit</zorder>
   <zorder>SexBox</zorder>
   <zorder>IDEdit</zorder>
   <zorder>BirthEdit</zorder>
  </widget>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>buttonBox</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>StuDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>StuDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
